src/components/Card/Card.js

```diff
@@ -25,12 +25,12 @@ class Card extends Component {
       <div className='card' id={this.state.id}>
         <img src={this.state.image} alt={this.state.name} />
         <h3>{this.state.name}</h3>
-        <p>{this.state.date}</p>
-        <p>{this.state.pay}</p>
-        <p>{this.state.fluff}</p>
-        {this.state.isBooked && <p>Job booked</p>}
-        {!this.state.isBooked && <button onClick={() => this.bookJob()}>Book Job 🚀</button>}
-        <button onClick={() => this.state.deleteJob(this.state.id)}>🗑</button>
+        <p className='date'>{this.state.date}</p>
+        <p className='pay'>{this.state.pay}</p>
+        <p className='fluff'>{this.state.fluff}</p>
+        {this.state.isBooked && <p className='booked'>Job booked</p>}
+        {!this.state.isBooked && <button className='book-btn' id={`book${this.state.id}`} onClick={() => this.bookJob()}>Book Job 🚀</button>}
+        <button className='delete-btn' id={`delete${this.state.id}`} onClick={() => this.state.deleteJob(this.state.id)}>🗑</button>
```

Comment:
This is a cool way to dynamically differentiate between the different button ids